### PR TITLE
465 responsive result browser

### PR DESCRIPTION
**Pull request by js-xc**
First version, let's start with this and add more when needed. 
So far: 
* bigger font in menu
* column layout in mobile/portrait mode
* filter menu made usable on mobile (full screen overlay)
* navigation can be expanded/hidden on mobile in portrait mode
* realbrowser action screenshots are sized to 100% screen width per default, zoomable

### CODE REVIEW

**Review by rschwietzke — APPROVED** [review]
Verified the demo. Approved.

File: resultbrowser/src/css/default.css [hunk 1]
```diff
@@ -627,3 +632,242 @@ table.key-value-table td.value .csep {
...
+	flex-flow: row;
+	width: 100vw !important;
+}
+
+#header .expander {
+	display: none;
+}
+
+#menu-close {
+	display: none;
```
Comment (jowerner) [inline resultbrowser/src/css/default.css:678]: How about adding `cursor: pointer;`?

Comment (js-xc) [reply resultbrowser/src/css/default.css:678]: To which element? `#menu-close` is only ever visible in mobile mode, where you don't see a cursor.

Comment (jowerner) [reply resultbrowser/src/css/default.css:678]: Haha, true!

**Pull request merged**